end-to-end-test/remote/specs/core/genomicEvolution.spec.js

```diff
@@ -101,5 +100,16 @@ describe('Patient View Genomic Evolution tab', function() {
                 'should show all mutations again, since none explicitly selected'
             );
         });
+
+        it('force sequential mode when samples have no equivalent clincal events', () => {
+            goToUrlAndSetLocalStorage(
+                `${CBIOPORTAL_URL}/patient/genomicEvolution?studyId=nsclc_ctdx_msk_2022&caseId=P-0016223`
+            );
+            expect(() =>
```

Comment:
this is a bit problematic because sometimes the page might load more slowly and this will fail too quickly and be "flakey".   a better way to test it would be to wait for another nearby sibling/parent of this element and which ALWAYS loads to display, and then test to see if this one also appears.     A good candidate in this case might be it's parent "data-test="VAFChartControls".   If the parent is displayed, then it should also be displayed.  So if you wait for the parent, which always displays, you can then test for the sequential mode toggle.